### PR TITLE
chore: move shared features to desktop

### LINKED ISSUE #5045: [Task]: Move shared `features` to desktop
### Task description

The features should be moved.

### Requirements

- [x] Move `features` to desktop

### Creation checklist

- [X] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by jeeanribeiro**
## Summary
This PR moves shared features folder to desktop and updates imports from it.
Extra: updates features imports for some mobile files by using its alias

## Changelog
```
- Moves shared features to desktop
- Updates features imports
```

## Relevant Issues
closes #5045

### Platforms
- __Desktop__
  - [ ] MacOS
  - [x] Linux
  - [ ] Windows
- __Mobile__
  - [ ] iOS
  - [ ] Android

## Checklist
- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/components/SettingsMenu.svelte [hunk 1]
```diff
@@ -11,7 +11,7 @@
     } from '@core/router'
     import { SETTINGS_ICON_SVG } from '@lib/auxiliary/icon'
     import { Icon, Text, TextType } from 'shared/components'
-    import features from 'shared/features/features'
+    import features from '../../desktop/features/features'
```
Comment (maxwellmattryan) [inline packages/shared/components/SettingsMenu.svelte:14]: Hmm I'm wondering if this should be inside the desktop components folder.

File: packages/shared/components/atoms/buttons/SendButton.svelte [hunk 2]
```diff
@@ -6,7 +6,7 @@
     import { appSettings } from '@core/app'
     import { Text, FontWeight, TextType, Icon, Modal } from 'shared/components'
     import { Icon as IconEnum } from '@auxiliary/icon'
-    import features from '@features/features'
+    import features from '../../../../desktop/features/features'
```
Comment (maxwellmattryan) [inline packages/shared/components/atoms/buttons/SendButton.svelte:9]: Same as above ^^

File: packages/shared/components/organisms/AccountActivity.svelte [hunk 3]
```diff
@@ -9,7 +9,7 @@
         setAsyncStatusOfAccountActivities,
     } from '@core/wallet'
     import { ActivityTile, Text, TextInput, TogglableButton, Filter, FontWeight } from 'shared/components'
-    import features from 'shared/features/features'
+    import features from '../../../desktop/features/features'
```
Comment (maxwellmattryan) [inline packages/shared/components/organisms/AccountActivity.svelte:12]: Same as above ^^

File: packages/desktop/electron/lib/menu.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { app, ipcMain, Menu, shell } from 'electron'
-import { DISCORD_URL, DOCUMENTATION_URL, FAQ_URL, ISSUE_REPORT_URL } from 'shared/lib/contexts/settings/constants'
-import features from 'shared/features/features'
+import { DISCORD_URL, DOCUMENTATION_URL, FAQ_URL, ISSUE_REPORT_URL } from '@contexts/settings/constants'
+import features from '@features/features'
```
Comment (maxwellmattryan) [inline packages/desktop/electron/lib/menu.js:3]: Do these work? I'm not sure how these are working in JS if they're specified for TS.

**Review by maxwellmattryan — CHANGES_REQUESTED** [review]
Some minor things, mainly depending on code inside of the desktop directory from shared, which we should try to avoid. Thanks for making this change man :fire:

File: packages/desktop/electron/lib/menu.js (hunk 4 above)
Comment (jeeanribeiro) [reply packages/desktop/electron/lib/menu.js:3]: I've tested and is working, maybe because of webpack resolve.alias configs

Comment (maxwellmattryan) [reply packages/desktop/electron/lib/menu.js:3]: Ah yes, probably that :+1:

File: packages/desktop/views/dashboard/wallet/Wallet.svelte [hunk 5]
```diff
@@ -1,8 +1,9 @@
 <script lang="typescript">
     import { selectedAccount } from '@core/account'
     import features from '@features/features'
-    import { ReceiveAddressButton, SendButton } from 'shared/components/atoms'
-    import { AccountActivity, AccountAssetsList, AccountSummary, Pane } from 'shared/components'
+    import { ReceiveAddressButton } from 'shared/components/atoms'
+    import { AccountActivity, SendButton } from '@components'
```
Comment (maxwellmattryan) [inline packages/desktop/views/dashboard/wallet/Wallet.svelte:5]: Can these two imports be combined?

**Review by maxwellmattryan — APPROVED** [review]
LGTM :fire:

**Pull request merged**